nameko/rpc.py

```diff
@@ -350,10 +351,16 @@ def __call__(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply.result()
 
-    def async(self, *args, **kwargs):
+    def call_async(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply
 
+    def async(self, *args, **kwargs):  # pragma: no cover
```

Comment:
@davidszotten Is there way to run tests locally? I can't find any explanation in the docs. It clearly expected that rabbitmq should be running on localhost, not sure how to pass it different amqp uri